pages/realms/new.tsx

```diff
@@ -146,13 +151,16 @@ const New = () => {
       const mint = await tryGetMint(connection.current, mintPublicKey)
       if (mint) {
         const supply = mint.account.supply
-        const decimals = mint.account.decimals
         if (supply.gt(new BN(0))) {
           handleSetForm({
             minCommunityTokensToCreateGovernance: BN.max(
               new BN(1),
-              // divide by 10^(decimal + 2) to get 1% of supply
-              supply.div(new BN(10).pow(new BN(decimals + 2)))
+              // divide by 100 for a percentage
+              new BN(
+                getMintDecimalAmount(mint.account, supply)
+                  .dividedBy(100)
+                  .toString()
+              )
```

Comment:
didnt shiftedBy before since it wasnt on BN but was on BigNumber haha

So I used units.ts for getting the BigNumber decimal amount and dividing but then I have to convert it back to BN unless we want to use BigNumber everywhere. Then there is the serum/anchor "BN" and the regular "BN" as well haha

Reply:
sticking to regular BN for now since thats what most of the API layer uses rn